### PR TITLE
Fix lints add badges

### PR DESCRIPTION
**Pull request by vadixidav**
Lints are temporarily fixed by allowing the specific unused code to pass CI. Generally we should try to ensure that things are only added if they are used and tested going forwards, but for now this is acceptable since it is known this will be needed.

### CODE REVIEW

File: crates/longboy/src/proto/sender.rs [hunk 1]
```diff
@@ -74,8 +73,7 @@ where
                 self.flags[index] = false;
             }
         }
-        if !self.flags[index]
-        {}
+        let _flag_set = !self.flags[index];
```
Comment (rickvanprim) [inline crates/longboy/src/proto/sender.rs:76]: This is either left over and should be deleted or a bug because something isn't checking the flag.

Comment (vadixidav) [reply crates/longboy/src/proto/sender.rs]: That is good. It was bothering me, and I wasn't quite sure why it was there. Initially I added a `todo!()`, but that caused the tests to fail.

Comment (rickvanprim) [reply crates/longboy/src/proto/sender.rs]: I suspect it was the initial form of checking whether or not to actually transmit something and was leftover.

**Review by rickvanprim — APPROVED** [review]

**Comment by vadixidav** [comment]
@rickvanprim You may as well merge it. I don't have permission to myself currently, which is fine.

**Pull request merged**